operations/iuf/workflows/management_rollout.md

```diff
@@ -23,6 +23,21 @@ Once this step has completed:
 
 ## 2. Execute the IUF `management-nodes-rollout` stage
 
+### Selective iSCSI worker node personalization
```

Comment:
These 2 new subsections need to be added to the table of contents at the top of the file.